### DISCUSSION TITLE
[BUG] Unable to clone github repo

### DISCUSSION THREAD

**Discussion by nao-hoshino** [zeapo/Android-Password-Store#3173] [Q&A]
### Describe the bug

When attempting to clone an existing repo (created previously using gopass) from Github, the process fails with an "exhausted available authentication methods" error:

![Screenshot_20240817-073054](https://github.com/user-attachments/assets/70d2b6f7-bbf9-49e9-a0da-9c0d7dfcbed9)

### Steps to reproduce

1. Choose option to clone a remote repo
2. Select SSH key authentication
3. Press 'Clone' button
4. App prompts for repository password
5. After entering the password, the aforementioned error occurs.

### Expected behavior

Finish setup without encountering any errors.

### Device information

- Device: Google Pixel 4
- OS: GrapheneOS
- App version: 1.13.5

**Comment by msfjarvis** [comment]
Try `git@github.com:nao-hoshino/gopass` instead
